src/shared/api/api.types.ts

```diff
@@ -54,3 +54,13 @@ export type RefreshResponse = {
   refreshToken: string;
   tokenType: string | null;
 };
+
+export enum TargetExtension {
+  MP3 = '.mp3',
```

Comment:
I didn't notice where MP3 was used?

Reply:
MP3 is used in the Audio Player Activity Item for recording audio files. By default webm is converting to the mp3.

![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/116339462/c174a942-bc17-429d-adb4-79aa97a2c049)